abnf/odata-abnf-testcases.yaml

```diff
@@ -2456,6 +2497,66 @@ TestCases:
     Rule: select
     Input: $select=*
 
+  - Name: 5.1.3 Select - $count
+    Rule: select
+    Input: $select=Sizes($count=true)
+
+  - Name: 5.1.3 Select - $count
+    Rule: select
+    Input: $select=Addresses($count=true)
+
+  - Name: 5.1.3 Select - $count
+    Rule: select
+    Input: $select=Sizes/$count
+
+  - Name: 5.1.3 Select - $count
+    Rule: select
+    Input: $select=Addresses/$count
+
+  - Name: 5.1.3 Select - $count
+    Rule: select
+    Input: $select=Sizes/$count($filter=$it gt 5)
+
+  - Name: 5.1.3 Select - $count
+    Rule: select
+    Input: $select=Addresses/$count($filter=startswith(City,'H'))
+
+  - Name: 5.1.3 Select - $count - no $ prefix for query options, still required for
```

Comment:
The input doesn't have query options, so why mention them?

Reply:
Fixed.